File: c/Lista1/Desafio4.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){

    int A, B, QA, QB, soma1, soma2;
    
    printf("Digite o número A: ");
    scanf("%i", &A);printf("Digite o número B: ");
    scanf("%i", &B);QA = pow (A,2);
    
    QB = pow (B,2);
    soma1 = A + B;soma2 = QA + QB;
    printf("A soma dos números A e B são: %i", soma1);
    printf("\n");
printf("A soma dos quadrados dos números de A e B são: %i", soma2);
    }

File: js/lista1/Desafio4.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

/*
Desafio4- Construa um programa que:

Leia 2 números inteiros A e B;

Calcule o quadrado para cada um;

Somar os dois;

Mostrar os resultados.
*/
 
rl.question('Informe o valor de A: ', (a) => {
    rl.question('Informe o valor de B: ', (b) => {
      
      const quadradoA = Math.pow(a,2);
      const quadradoB = Math.pow(b,2);
      const soma = quadradoA + quadradoB;
      
      console.log(`A soma do quadrado A(${quadradoA}) + quadrado B(${quadradoB}): ${soma}`);
      rl.close();
    });
  });

